### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by lazasa**

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -1,11 +1,23 @@
...
   app.useGlobalPipes(new ValidationPipe())
+  
+  const config = new DocumentBuilder()
+  .setTitle('API')
+  .setDescription('ARB Practice')
+  .setVersion('1.0')
+  .addTag('ARB')
+  .build()
+  const document = SwaggerModule.createDocument(app, config)
+  SwaggerModule.setup('api', app, document)
```
Comment (martinalbrnz) [inline src/main.ts:18]: your linter will give me a headache

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "editor.codeActionsOnSave": {
-    "source.fixAll": true,
-    "source.organizeImports": true
+    "source.fixAll": "explicit",
+    "source.organizeImports": "explicit"
```
Comment (martinalbrnz) [inline .vscode/settings.json:4]: what a change bro 😎

**Review by martinalbrnz — APPROVED** [review]
seems like u got an approved pr, good job 😎

File: src/main.ts (hunk 1 above)
Comment (lazasa) [reply src/main.ts:18]: es especial

File: .vscode/settings.json (hunk 2 above)
Comment (lazasa) [reply .vscode/settings.json:4]: breaking change

Comment (martinalbrnz) [reply .vscode/settings.json:4]: rip our beloved api, press F to pay respects

**Pull request merged**